### ISSUE TITLE
Hi there,  Something is wrong with the Loss...

### ISSUE THREAD

**Issue by WenWeiZhao** [aiff22/PyNET#13]
Hello, Why is the calculation of loss not consistent with the paper, and the weight of each part of loss is not the same?  The tensorflow version.  
Look forward to your reply. Thanks.

**Comment by aiff22** [comment]
Hi @WenWeiZhao, the answer was provided [in this thread](https://github.com/aiff22/PyNET-PyTorch/issues/1).
